app/javascript/src/components/_map.scss

```
@@ -28,12 +28,21 @@
     bg-gray-500
   ;
 
-  --zoom-amount: 1.5;
+  --drawer-scale: 1.25;
 
   .token {
     @apply
       static
       m-2
     ;
+
+    .token__image {
+      height: calc(var(--base-size) * var(--drawer-scale));
+      width: calc(var(--base-size) * var(--drawer-scale));
+    }
+
+    .token__name {
+      margin-left: calc((var(--base-size) + 4px));
```

The names have started overlapping with the token in the drawer

```suggestion
      margin-left: calc((var(--base-size) + 4px) * var(--drawer-scale));
```